protocol/activity_center.go

```diff
@@ -15,9 +15,11 @@ import (
 type ActivityCenterType int
 
 const (
-	ActivityCenterNotificationTypeNewOneToOne = iota + 1
+	ActivityCenterNotificationNoType = iota
+	ActivityCenterNotificationTypeNewOneToOne
 	ActivityCenterNotificationTypeNewPrivateGroupChat
 	ActivityCenterNotificationTypeMention
+	ActivityCenterNotificationTypeReply
 )
```

Comment:
I see that these consts are used as enums of the `ActivityCenterType` type but there is no type assertion on the consts.

Easily implemented though, see below:

```go
const (
	ActivityCenterNotificationNoType ActivityCenterType = iota
	ActivityCenterNotificationTypeNewOneToOne
	ActivityCenterNotificationTypeNewPrivateGroupChat
	ActivityCenterNotificationTypeMention
	ActivityCenterNotificationTypeReply
)
```